tutorial/cdn-caching.md

```diff
@@ -0,0 +1,158 @@
+---
+title: CDN Caching
+nav_order: 22
+parent: Malibu Tutorial
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Harshith](ttps://www.linkedin.com/in/harshith-raj-092ba4176)_
+
+How Quintype handles CDN caching?
+
+- Whenever a collection/story/author is created or updated, the bold handles cache/purging through the CDN.
+- When a collection/story/author is created or updated, the frontend gets updated with the latest data with the help of cache-keys.
+
+How cache keys are created
+
+1. In Quintype a set of cache keys are generated by bold when a story/collection API response is being returned. The format for the cache keys remains the same when creating from the app level.
+
+    Eg:
+    ![Register Screen]({{"images/route-data-json.png" | absolute_url}})
+2. In frontend we concat additional cache keys with the cache key generated by bold in quintype-node-framework (server/caching.js). The logic of creating cache keys is explained below
+3. Format for home-page:
+
+   ```javscript
+   `{page/${publisherId}/home-page}`
+   ```
+
+   We concat the collection cache keys, story cache keys, section collection cache keys to the above-generated key. The front end gets updated after it sees the cache keys from the bold when the home collection gets updated and published.
+
+   Eg:
+
+   `< cache-tag: c/1166/87520, c/1166/96964, s/1166/5b10a46a, s/1166/185a3cb0, page/1166/home-page`
+
+   Note:
+
+   - We form the cache keys for nested collection by going from the top-level depth of the collections .i.e, for example, the home collection contains 2 collections and each of these collections has a story then based on the above example `c/1166/87520` is generated based on the first collection in home, `c/1166/96964`is generated based on the second collection in home, `s/1166/5b10a46a` is generated based on the story in the first collection and so on
```

Comment:
Specify the default nesting depth, don't remember, its 1 or 2.

Reply:
sure

Reply:
@rashmirmittal, is this required we are just mentioning how keys are being formed w.r.t nested collection

Reply:
Required because there is a default depth and custom FEs can override the depth